.github/workflows/comment-on-asciidoc-changes.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Comment on PR for .asciidoc changes
+
+on:
+  pull_request:
+    types:
+      - synchronize
+      - opened
+      - reopened
+    branches:
+      - main
+      - master
+      - "9.0"
```

Comment:
There's no master branch on this repo, and it doesn't follow the ES releases, so there's no 9.0 branch either.

```suggestion
```